WcaOnRails/.rubocop_todo.yml

```diff
@@ -234,6 +234,7 @@ Style/ClassAndModuleChildren:
     - 'app/controllers/accounts/registrations_controller.rb'
     - 'app/controllers/api/v0/api_controller.rb'
     - 'app/controllers/api/v0/competitions_controller.rb'
+    - 'app/controllers/api/v0/records_controller.rb'
```

Comment:
I like the look of the so-called "compact" style. I'd support changing this rubocop [`Style/ClassAndModuleChildren` setting](https://github.com/bbatsov/rubocop/blob/e4d03d70853ef54ce6578fd84da83763a8830474/config/default.yml#L309-L328) from `nested` to `compact`. @larspetrus, @timhabermaas, @jonatanklosko, as our more experienced rubyists, how do you feel about this change? Is there something evil about the nested module syntax that we should avoid it?